### PR TITLE
Course Builder v3: same page load from backend and frontend dashboard.

### PR DESCRIPTION
**Pull request by harunollyo**

### CODE REVIEW

File: views/pages/course-builder.php [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+	<meta charset="UTF-8">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<title><?php esc_html_e( 'Tutor Course Builder', 'tutor' ); ?></title>
+</head>
+<body>
+
+	<div id="tutor-course-builder">React App Here</div>
+	<!-- scripts -->
+	<?php //phpcs:ignore WordPress.WP.EnqueuedResources.NonEnqueuedScript ?>
+	<script src="<?php echo esc_url( tutor()->url . 'assets/js/tutor-course-builder-v3.min.js?v=' . TUTOR_VERSION ); ?>"></script>
```
Comment (shewa12) [inline views/pages/course-builder.php:29]: Why enqueueing here?

File: classes/Course.php [hunk 2]
```diff
@@ -198,6 +198,63 @@ public function __construct() {
...
+		// Edit mode.
+		if ( $backend_edit || ( $has_pro && $frontend_edit ) ) {
+			$course_id        = Input::get( 'post' ) ?? Input::get( 'course_id' ) ?? 0;
+			$post_type        = get_post_type( $course_id );
+			$course_author    = (int) get_post_field( 'post_author', $course_id );
+			$is_course_author = get_current_user_id() === $course_author;
+
+			if ( tutor()->course_post_type === $post_type && ( User::is_admin() || $is_course_author ) ) {
+				$this->load_course_builder_view();
+			}
```
Comment (shewa12) [inline classes/Course.php:242]: We need to a `unauthorized template` that will be loaded if `$backend_create || ( $has_pro && $frontend_create )` &&  `$backend_edit || ( $has_pro && $frontend_edit )` both conditions failed

**Review by shewa12 — CHANGES_REQUESTED** [review]

File: classes/Course.php (hunk 2 above)
Comment (harunollyo) [reply classes/Course.php:240]: Access control has been enhanced.

Unauthorized templates are no longer necessary. In the backend, WordPress will handle it automatically, and on the frontend, it is already managed by the tutor.

File: views/pages/course-builder.php (hunk 1 above)
Comment (harunollyo) [reply views/pages/course-builder.php]: Now it's enqueuing natively.

**Review by shewa12 — APPROVED** [review]

**Pull request merged**